EEG Game/Assets/Scripts/Input/InputManager.cs

```diff
@@ -0,0 +1,55 @@
+using UnityEngine;
+using UnityEngine.InputSystem;
+
+public class InputManager : MonoBehaviour
+{
+    [Header("Input Settings")]
+    public bool toggleCrouch;
+
+    [Header("Inputs")]
+    public Vector2 movement;
+    public bool jump;
+    public bool crouch;
+
+    private static InputManager instance;
+
+    public static InputManager Instance
+    {
+        get { return instance; }
+    }
+
+    private void Awake()
+    {
+        // Make InputMager a Singleton
```

Comment:
"InputMager" typo?

Reply:
Oh my bad let me fix that also yee I'll look into mouse sensitivity real quick

Reply:
I think we can keep the dragonslayer assets for testing